### DISCUSSION TITLE
ADIF import from command line?

### DISCUSSION THREAD

**Discussion by m0urs** [magicbug/Cloudlog#1612] [Q&A] [answered]
Is there a possibility to import ADIF files via command line? The use case for that would be that I do have programs like VarAC, whcih write ADIF files but cannot directly log to Cloudlog. So I would like to run a task every night importing the ADIF file into Cloudlog (which then just imports the new QSOs and skips the duplicates).

**Comment by magicbug** [comment, marked as answer]
Simplest solution is to utilise the API you can then send QSOs from anything, however I think your asking if there's a premade tool which currently there isn't officially.

There is some tools on Github though see https://github.com/search?q=cloudlog+adif&type=Repositories

**Reply by m0urs** [reply]
Thank you very much. I will check this out.

**Comment by phl0** [comment]
This one by @m1geo looks promising: https://github.com/m1geo/Cloudlog-ADIF-Uploader

**Reply by m0urs** [reply]
Thank you very much. I will check this out.

**Reply by m0urs** [reply]
Thanks, I will use this python script now. It is very simple and does exactly what I wanted. I run it every day at night via the windows task scheduler.
